crates/project_panel/src/project_panel_tests.rs

```diff
@@ -5821,6 +5821,103 @@ async fn test_hide_root(cx: &mut gpui::TestAppContext) {
     }
 }
 
+#[gpui::test]
+async fn test_compare_selected_files(cx: &mut gpui::TestAppContext) {
+    init_test_with_editor(cx);
+
+    let fs = FakeFs::new(cx.executor().clone());
+    fs.insert_tree(
+        "/root",
+        json!({
+            "file1.txt": "content of file1",
+            "file2.txt": "content of file2",
+            "dir1": {
+                "file3.txt": "content of file3"
+            }
+        }),
+    )
+    .await;
+
+    let project = Project::test(fs.clone(), ["/root".as_ref()], cx).await;
+    let workspace = cx.add_window(|window, cx| Workspace::test_new(project.clone(), window, cx));
+    let cx = &mut VisualTestContext::from_window(*workspace, cx);
+    let panel = workspace.update(cx, ProjectPanel::new).unwrap();
+
+    cx.simulate_modifiers_change(gpui::Modifiers {
+        control: true,
```

Comment:
That would not work on mac, and I'm not sure what happens here at all, why do we do this?

Reply:
Any action we call is the same action that's usually called somewhere based on a keybinding/mouse events.
So, if we call an action, we sort of assume that it happened after some cmd-click or ctrl-shift-a or whatever.

Reply:
For some stupid reason, had to come and remove this myself finally...